pkg/cloud/aws/metadata.go

```diff
@@ -42,11 +43,28 @@ func (m *MetadataProvider) GetMetadata() map[string]string {
 	sess.ClientConfig(s3.ServiceName)
 
 	stsClient := sts.New(sess)
-	identity, err := stsClient.GetCallerIdentity(&sts.GetCallerIdentityInput{})
+
+	const (
+		maxInterval    = 200 * time.Millisecond
+		maxElapsedTime = 3 * time.Second
```

Comment:
total 3 seconds made the difference?

Reply:
It's not about the time, it's more about the tries. In the playground we'll only start lakeFS if it (`aws sts get-caller-identity`) succeeded once before. But since AWS IAM is eventual consistent, there is a chance that it will fail from here. Retrying it here at least 15 times makes it much more reliable to get at least one successful response.